.gherkin-lintrc

```diff
@@ -5,7 +5,7 @@
   "no-dupe-scenario-names": "on",
   "no-dupe-feature-names": "on",
   "no-partially-commented-tag-lines": "on",
-  "indentation": "on",
+  "indentation": ["on"],
```

Comment:
You don't need the bracket anymore. In fact it will raise an error in rule.js because when the configuration is an array it's expected to be of length 2, the first element is the "on"/"off" switch and the second element is the rule custom configuration fields.